### ISSUE TITLE
Generation fails after upgrade to 15.1.2

### ISSUE THREAD

**Issue by Garbee** [angular/angular-cli#24586]
### Command

generate

### Is this a regression?

- [X] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

15.1.1

### Description

Upon generating a new component, an exception is thrown from the CLI tool. I traced this to the code touched from: https://github.com/angular/angular-cli/pull/24542. The `app-shell` schematic in particular is what was undefined, thus causing the error.

This oddly **does not** present itself on a freshly made application stack. Only from a stack that was upgraded from 15.0. (Could happen when upgrading from older versions as well.)

For now, I have rolled `@angular/cli` back by force to `15.1.1` and the problem is resolved since that includes the older version of the devkit.

### Minimal Reproduction

1. Have an application on version 15.0
2. Upgrade to 15.1 using `ng update` on all appropriate dependencies (or just the CLI)
3. Attempt to run `ng g c test`

### Exception or Error

```text
An unhandled exception occurred: Cannot destructure property 'aliases' of 'collection.description.schematics[schematicName]' as it is undefined.
```

### Your Environment

```text
_                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 15.1.2
Node: 18.12.1
Package Manager: npm 9.2.0
OS: darwin x64

Angular: 15.1.1
... animations, cdk, common, compiler, compiler-cli, core, forms
... localize, material, platform-browser
... platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1501.2
@angular-devkit/build-angular   15.1.2
@angular-devkit/core            15.1.2
@angular-devkit/schematics      15.1.2
@angular/cli                    15.1.2
@schematics/angular             15.1.2
rxjs                            7.8.0
typescript                      4.9.4
```

**Comment by alan-agius4** [comment]
Thanks for opening this issue. 

Fortunately this has been already solved via https://github.com/angular/angular-cli/pull/24575 and will be part of 15.1.3

**Comment by ZumelzuR** [comment]
> Thanks for opening this issue.
> 
> Fortunately this has been already solved via #24575 and will be part of 15.1.3

what we can do meanwhile this version is released ?

**Comment by alan-agius4** [comment]
Downgrade to 15.1.1
